modules/gapi/include/opencv2/gapi/gmetaarg.hpp

```diff
@@ -61,6 +61,16 @@ namespace detail
 
 } // namespace detail
 
+// Forward declaration for gmat
+class Mat;
+class UMat;
+GAPI_EXPORTS std::vector<cv::GMetaArg> descr_of(const std::vector<cv::Mat> &vec);
```

Comment:
please use for concistency with other version of `descr_of` , i.e. 
```suggestion
GAPI_EXPORTS cv::GMetaArgs  descr_of(const std::vector<cv::Mat> &vec);
```